composer.json

```diff
@@ -15,7 +15,8 @@
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/seo-bundle": "^1.1.3|^2.0",
-        "doctrine/collections": "^1.0"
+        "doctrine/collections": "^1.0",
+        "phpcr/phpcr-utils": "^1.2.9"
```

Comment:
php 7 now works. 7.1 keeps having the error, but says it installed 1.2.9 so i really don't know how that can happen. lets try again tomorrow. (here in vancouver, its only 16:30 :-) )